src/React.re

```diff
@@ -872,10 +872,15 @@ module Uncurried = {
     "useCallback";
 };
 
+type callback('input, 'output) = 'input => 'output;
```

Comment:
this is already defined in line 471

Reply:
just saw https://github.com/reasonml/reason-react/commit/757dd23e1b1c22a2fc82fd48b4951c65e84235d2

Reply:
Sorry for confusing - This PR should be PASSED Pipelines before merge. @anmonteiro